src/index.js

```diff
@@ -32,6 +32,11 @@ app.get('api/user', (req, res) => {
     return res.json(user_id)
 })
 
+app.get('/api/product', (req, res) => {
+    const {prod_id} = req.query;
```

Comment:
is this product id is unique?
and it only returning that product id.

Reply:
yes. product id should be unique.
working on this route